.github/workflows/deploy.yml

```
@@ -0,0 +1,17 @@
+name: Deploy
+on:
+  push:
+    tags:
+    - "*"
+  workflow_run:
+    workflows:
+      - "Package"
+    types:
+      - completed
+jobs:
+  notify:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

`${{}}`なしでいいんじゃないかな。

```suggestion
    if: github.event.workflow_run.conclusion == 'success'
```